### DISCUSSION TITLE
Color-coded actions + separators

### DISCUSSION THREAD

**Discussion by vesper8** [laravel/nova-issues#4958] [Ideas & Feature Requests]
In my use case my actions dropdown is getting a bit out of control with over 25 actions.

It would be great to be able to pass a hex colour (or css class) to each action. Similarly it would be nice to be able to add separators and add "action groups"

I know both of these are possible since there is a separator between replicate/force delete and the rest of the custom actions.. and the force delete action is in red.

Many thanks

**Comment by juse-less** [comment]
With or without many actions, being able to group related ones would be a really great addition.
My users aren't too technical, so being able to group them would help me make things easier to understand for them.
